### PR TITLE
C20_WMDE_iPad_Test_01

### PR DESCRIPTION
**Pull request by Abban**
https://phabricator.wikimedia.org/T266758
https://phabricator.wikimedia.org/T266759

### CODE REVIEW

File: pad/components/BannerText.jsx [hunk 1]
```diff
@@ -3,23 +3,28 @@ import { h } from 'preact';
...
 		<p className="text__headline text__headline--bold">
-			<span className="text__headline--italic">Liebe Leserinnen und Leser, </span>
+			<img className="info-icon" src="https://upload.wikimedia.org/wikipedia/commons/9/93/Info-icon-black-on-yellow.svg" alt="info_icon" width="16" height="16" />
+			<span className="text__headline--italic"> Liebe Leserinnen und Leser, </span>
 			<span>verzeihen Sie die Störung. <span> { capitalizeFirstLetter( weekdayPrepPhrase ) } { currentDayName } sind Sie in Deutschland gefragt:</span></span>
 		</p>
 
 		<p className="text__paragraph text__paragraph--bold">
-			<span>{ campaignDaySentence } Wikipedia wird durch Spenden von durchschnittlich 23,83&nbsp;€ finanziert, aber
+			<span>{ campaignDaySentence } Wikipedia wird durch Spenden von durchschnittlich 23,81&nbsp;€ finanziert, aber
```
Comment (moiikana) [inline pad/components/BannerText.jsx]:
```suggestion
			<span>{ campaignDaySentence } Wikipedia wird durch Spenden von durchschnittlich 22,81&nbsp;€ finanziert, aber
```

File: pad/components/BannerText.jsx [hunk 2]
```diff
@@ -3,23 +3,28 @@ import { h } from 'preact';
...
 				99&nbsp;% der Leserinnen und Leser spenden nicht. </span>
 
 			<span className="text__highlight">Wenn alle, die das jetzt lesen, einen kleinen Beitrag leisten, wäre
 				unser Spendenziel bereits am heutigen { currentDayName } erreicht.</span>
 
 			<span> Schon der Preis einer Tasse Kaffee würde genügen.
-			Über { campaignParameters.millionImpressionsPerDay } Millionen Mal wird unser Spendenaufruf täglich angezeigt, aber
-			nur { numberOfDonors } Menschen haben bisher gespendet.
+			Über { daysSinceCampaignStart >= 2 ? campaignDonationSentence : '' }
```
Comment (moiikana) [inline pad/components/BannerText.jsx]: needs a space after the sentence

File: pad/components/BannerText.jsx (hunk 1 above)
Comment (Abban) [reply pad/components/BannerText.jsx]: Oh, I probably made this mistake in every banner I worked on

**Comment by moiikana** [comment]
rebased to master,
still needs new use of funds

**Pull request merged**